gradle/libs.versions.toml

```diff
@@ -7,15 +7,15 @@ errorproneSlf4j = "0.1.18"
 googleJavaFormat = "1.17.0"
 guava = "32.0.0-jre"
 hadoop = "2.7.3" # this is in mapping with iceberg repo.
-hive = "2.3.8" # this is in mapping with iceberg repo.
-iceberg = "1.2.1"
+hive = "2.3.9" # this is in mapping with iceberg repo.
+iceberg = "1.3.0"
```

Comment:
nit: maybe add comment with link https://github.com/apache/iceberg/blob/apache-iceberg-1.3.0/versions.props so easy looking up the versions in the future

Reply:
since it contains the release version in the link it needs update each time. 
I will add the PR comments next time for easy review.